pkg/cmd/roachtest/tests/lease_preferences.go

```diff
@@ -188,7 +188,8 @@ func runLeasePreferences(
 				// dc=2: n3 n4
 				// ...
 				// dc=N: n2N-1 n2N
-				fmt.Sprintf("--locality=region=fake-region,zone=fake-zone,dc=%d", (node-1)/2+1))
+				fmt.Sprintf("--locality=region=fake-region,zone=fake-zone,dc=%d", (node-1)/2+1),
+				"--vmodule=replica_proposal=2,replicate_queue=3,replicate=3")
```

Comment:
Did you check that this won't be too noisy?

Reply:
I've looked at the logs few times with this change, whilst debugging the failure linked.

It's fine for cockroach.log logging, since there's no workload running. Turning it on after the initial splits/scatter would be better, but I'd rather have it all.